.integration-watchlist

```diff
@@ -4,12 +4,12 @@ src/
 integration/
 
 # exceptions
+!.eslintignore
```

Comment:
Just curious why do we add it this time?

Reply:
This is just an exception list of files we do not need to trigger integration tests when there are changes. eslintignore is not really needed to run integ tests so i added it.